code/style.css

```diff
@@ -1,4 +1,214 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #ffffff;
+  color: #000;
+  font-family: Arial, Helvetica, sans-serif;
+}
+body{
+margin: 0px;
+}
+
+.site-name{
+  text-align: center;
+}
+
+span{
+  font-size: 15px;
+}
+
+/* rules for the header */
+header{
+  border: solid;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.logo{
+  height: 70px;
+  width: 80px;
+  justify-self: flex-end;
+}
+
+ul{
+  list-style: none;
+  justify-self: start;
+}
+
+a{
+  color: black;
+  text-decoration: none;
+}
+
+/* hover effect for navbar */
+a:hover {
+color: tomato;
+}
+
+/* meda rule to dissapear the site name in header when shown in mobile */
+@media (max-width:767px){
+  .site-name{
+    display:none;
+  }
+}
+
+/* media rule to make navbar into a row when shown on screen */
+
+@media (min-width: 992px){
+  ul{display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+  }
+  li{
+    padding-left: 15px;
+  }
+}
+
+.main-heading{
+  text-align: center;
+}
+
+/* rules for the main section with a news article.*/
+
+.big-news{
+  display: flex;
+  text-align: center;
+}
+
+.article-heading{
+  text-align: left;
+  margin: 0% 3%;
+}
+
+.article{
+  text-align: left;
+  margin: 0% 3%;
+  }
+
+.main-content-img{
+  justify-content: center;
+  border: 1px solid;
+  padding: 1px;
+  box-shadow: 5px 10px lightgray;
+}
+
+.read-more{
+  font-style: italic;
+}
+
+.text{
+  margin-top: 5%;
+}
+
+  /* rules for main article in tablet and screen  */
+@media (min-width:768px){
```

Comment:
Here comes another tablet-media queries but with 768px instead of 767px - is there any reason for this?

Reply:
i was thinking that mobile size end at 767 so that tablet should start from 768, did i get that wrong? should it start at the same px size as the mobile ends on? 🤔